scripts/aws.sh

```diff
@@ -0,0 +1,488 @@
+#!/bin/bash
+
+set -uo pipefail
+
+export AWS_PAGER=""
+
+err_log="err.log"
+
+trap 'rm -f "$err_log"' EXIT
+
+declare -A ami_map=(
```

Comment:
Is this a duplication of info defined in openshift-release repo? 
We should not copy it as it'll get out of sync very quickly. 

I would suggest putting this into the [include file](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift/microshift/includes/openshift-microshift-includes-commands.sh). This way, we can `curl` the file and use it here.

Reply:
Actually, this would only work for a single region. It's good in CI case, but I'm not sure we should do it in this script. 
We should require region and AMI id as command line arguments to make it simpler.